### PR TITLE
feat: flash messages

### PR DESCRIPTION
**Pull request by didoda**
This provides basic flash messages on controllers.
Usage:
```
$this->Flash->info('This is an info');
$this->Flash->warning('This is a warning');
$this->Flash->error('This is an error');
$this->Flash->success('This is a success');
```

### CODE REVIEW

**Comment by stefanorosanelli** [comment]
Seems ok!
I would drop `success` for now and use only `info`, `warn`, `error` like in BE3

File: src/Controller/ModulesController.php [hunk 1]
```diff
@@ -125,6 +125,8 @@ public function save()
         $this->apiResponse = $this->apiClient->saveObject($this->objectType, $this->request->getData());
         // TODO: error if $this->apiResponse['data']['id'] empty or  $this->apiResponse['error'] not empty
 
+        $this->Flash->success('Object saved');
```
Comment (stefanorosanelli) [inline src/Controller/ModulesController.php]: what about i18n?
```
$this->Flash->success(__('Object saved'));
```
same applies to other messages

File: src/Controller/TrashController.php [hunk 2]
```diff
@@ -83,7 +83,9 @@ public function view($id)
      */
     public function restore()
     {
-        $this->apiResponse = $this->apiClient->restoreObject($this->request->getData('id'));
+        $this->apiResponse = $this->apiClient->restoreObject($this->request->getData('id'), $this->request->getData('object_type'));
+
+        $this->Flash->success('Object restored');
```
Comment (stefanorosanelli) [inline src/Controller/TrashController.php]: I would use info for now, like
```
 $this->Flash->info(__('Object restored'));
```

**Comment by didoda** [comment]
After last commit, usage:
```
$this->Flash->info('This is an info');
$this->Flash->warning('This is a warning');
$this->Flash->error('This is an error');
```

**Pull request merged**